cabal-install/Distribution/Solver/Types/PackageConstraint.hs

```diff
@@ -97,6 +99,8 @@ data PackageProperty
    | PackagePropertySource
    | PackagePropertyFlags     FlagAssignment
    | PackagePropertyStanzas   [OptionalStanza]
+  -- MAYBE if we want to filter out packages without the specified sublibs,
+  -- add a PackagePropertySublibraries
```

Comment:
Nah, that seems like a silly thing to filter for :)

Reply:
Removing it